images/Dockerfile.py3.aws

```diff
@@ -0,0 +1,99 @@
+# Build the docker image used to run the scripts
+# to continuously update our docker files.
+#
+# The context for this docker file should be the root of the kubeflow/testing repository.
+FROM ubuntu:18.04
+
+RUN apt-get update -y && \
+    apt-get install -y curl git python3.8 python3-pip wget jq && \
+    ln -sf /usr/bin/python3.8 /usr/bin/python
+
+RUN python3.8 -m pip install \
+    filelock \
+    fire \
+    google-api-python-client \
```

Comment:
Yes, because those Google Packages are required in common python files.

Like `util.py` file, we not only have google packages but boto3, that means not only AWS but GCP's dockerfile needs to have all the necessary packages installed.